lib/killbill_client/models/invoice_payment.rb

```diff
@@ -69,6 +71,22 @@ def bulk_create(external_payment = false, user = nil, reason = nil, comment = ni
                             :comment => comment,
                         }.merge(options)
       end
+
+      def chargeback_reversal(payment_id, amount, adjustments = nil, user = nil, reason = nil, comment = nil, options = {})
+        payload             = InvoicePaymentTransactionAttributes.new
+        payload.amount      = amount
+        payload.is_adjusted = !adjustments.nil?
```

Comment:
This is not part of the API, is it?

Only a `transactionExternalKey` and an `effectiveDate` are required (https://github.com/killbill/killbill/blob/6b092a0ca5d812ac6acab938b2b874b59c9abcf9/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/InvoicePaymentResource.java#L257).